documentation/api/io.strimzi.api.kafka.model.connect.ExternalConfiguration.adoc

```diff
@@ -76,14 +77,30 @@ spec:
 [id='property-kafka-connect-external-volumes-{context}']
 === `volumes`
 
-You can also mount ConfigMaps or Secrets to a Kafka Connect pod as volumes.
+Use volumes to mount ConfigMaps or Secrets to a Kafka Connect pod.
 
 Using volumes instead of environment variables is useful in the following scenarios:
 
-* Mounting truststores or keystores with TLS certificates
 * Mounting a properties file that is used to configure Kafka Connect connectors
+* Mounting truststores or keystores with TLS certificates
+
+Configuration providers load the values from mounted ConfigMaps or Secrets.
```

Comment:
Being pedantic: In general config providers are a way to inject values from outside the configuration. Here we're using the provided config providers to load values from mounted ConfigMaps or Secrets. They could also be loaded from other files in the container, or with a some other config provider from somewhere else entirely.

Reply:
Changed to: _Configuration _providers_ load values from outside the configuration._
Also added: _Use a comma-separated list if you want to add more than one provider, including custom providers.
You can use custom providers to load values from other file locations._